### DISCUSSION TITLE
What exactly is the implication of not having the encrypted DNS records of the Paid Plans?

### DISCUSSION THREAD

**Discussion by NeeshSamsi** [forwardemail/forwardemail.net#191] [Q&A] [answered]
So the main selling point for upgrading to the $3 a month plan is the Encryption on DNS records so they're not publicly accessible. What exactly are the implications of this information being publicly available?

**Comment by titanism** [comment, marked as answer]
DNS records are public, so if you are on Enhanced Protection your forwarding addresses are hidden (they are not in DNS records, they're in a secure database).
